cns/service/main.go

```diff
@@ -1191,10 +1191,19 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 		if err != nil {
 			if errors.Is(err, store.ErrKeyNotFound) {
 				logger.Printf("[Azure CNS] No endpoint state found, skipping initializing CNS state")
+				if os.Getenv("StatelessCNIMigration") == "true" {
```

Comment:
can this come from the config instead of the env? and to align with the convention there, maybe`EnableManagedStateMigration`